local_setup.md

```diff
@@ -0,0 +1,193 @@
+# Local Setup
+
+For developing you want to set up the renuo-cms and the renuo-upload locally. They must be set up once and can then be used by every application that uses one or both of those services.
+
+## Step by Step Manual
+
+### Install renuo-cms-api
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-cms-api.git
```

Comment:
we should avoid redundancies in our documentations. this should already be described in the renuo-cms-api readme? i would link to that instead.